### ISSUE TITLE
Keyboard navigation -  On FF, it takes 5 tab presses to focus on the play button after leaving the header

### ISSUE THREAD

**Issue by LilyL0u** [bbc/simorgh#3709]
**Describe the bug**
When using the tab key to navigate the live radio player page, after the user has left the header elements, it takes 5 tab presses to focus on the play button. 

This only happens on FIREFOX

**To Reproduce**

1. Go to https://www.test.bbc.com/korean/bbc_korean_radio/liveradio
2. Use the tab button to navigate through the interactable web elements
3. Count the number of times you need to press tab to navigate to the play button

**Expected behavior**
The first focus after the header elements should be the play button.

**Desktop (please complete the following information):**

- OS: iOS
- Browser: Firefox
- Version: 69.0 (64-bit)

**Testing notes**
Given the user is on the page
And they are using keyboard navigation to interact with the page
And they press tab to get to the play button
Then they reach the play button one tab press after navigating away from the header elements
- [x] This bug fix is expected to need manual testing.

**Additional context**

**Comment by ryanmccombe** [comment]
https://jira.dev.bbc.co.uk/browse/SMPHTML-1457?focusedCommentId=4501212&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4501212

**Comment by gavinspence** [comment]
Fixed.
